docs/WritingTests.asciidoc

```diff
@@ -1277,8 +1277,25 @@ test module.
 
 ==== Enable snapshots for each module
 
-* Run the worker with --no-cleanup parameter. This will preserve the hard
- disks after test runs.
+* Run the worker with `--no-cleanup` parameter. This will preserve the hard
+ disks after test runs. If you are starting the worker(s) via the systemd unit,
+ then you can achieve this by creating the file
+ `/etc/systemd/system/openqa-worker@1.service.d/override.conf` with the
```

Comment:
Note that `systemctl edit openqa-worker@1` allows one to create that file without specifying the path. I suppose that also covers only the first worker and `systemctl edit openqa-worker@` would cover all.

Reply:
I'll try that out and update the PR.

Reply:
done